project-api/src/dal/Jalasoft.TeamUp.Projects.DAL/ProjectsRepository.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Jalasoft.TeamUp.Projects.DAL
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using Jalasoft.TeamUp.Projects.DAL.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+
+    public class ProjectsRepository : IProjectsRepository
+    {
+        private List<Health> healthList = new List<Health>() { new Health() { Message = "I'm resumes-api and I'm alive and running! ;)" } };
```

Comment:
please make this a only a Health Object instead of a list and change the message to "projects-api"